packages/catalog-search/package.json

```diff
@@ -23,7 +33,8 @@
   "license": "AGPL-3.0",
   "homepage": "https://github.com/edx/frontend-enterprise#readme",
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "directory": "dist"
```

Comment:
This config option tells Lerna to only publish the contents of the `dist` folder, rather than all the files. The `make build` command ensures all the necessary files are added to the `dist` folder at build time.